tests/pds/pds_common.go

```diff
@@ -658,24 +659,84 @@ func KillDbMasterNodeDuringStorageIncrease(dsName string, nsName string, deploym
 	return nil
 }
 
+// GetDeploymentsPodRestartCount to calculate pods restart count from deployment
+func GetDeploymentsPodRestartCount(deployment *pds.ModelsDeployment, namespace string) (int32, error) {
```

Comment:
are we using this method ?

Reply:
I was using it earlier, but not removing it for future use.